.mergify.yml

```diff
@@ -1,3 +1,10 @@
+queue_rules:
+  - name: default
+    conditions:
+      - status-success=run_build_pipeline
+      - base=master
```

Comment:
I don't think the `base=master` condition is necessary in the queue_rules (but nor is it doing any harm here)